### PR TITLE
Support bundling on windows

### PR DESCRIPTION
**Pull request by madtisa**
Current implementation of the plugin generates invalid paths on windows:
```js
globalThis.__bundlerPathsOverrides = {'pino/file': pinoWebpackAbsolutePath('.\path.js')...
```
instead of escaped paths:
```js
globalThis.__bundlerPathsOverrides = {'pino/file': pinoWebpackAbsolutePath('.\\path.js')...
```
This PR should fix this problem.
(similar to #102)

### CODE REVIEW

**Comment by simoneb** [comment]
@madtisa tests don't pass

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by mcollina** [comment]
Please exclude Windows & Node.js v14 combo on GHA. It's broken everywhere.

**Comment by simoneb** [comment]
@madtisa as per Matteo's comment, feel free to remove Node 14 from the build matrix in this same PR

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,12 +3,16 @@ name: CI
...
-    runs-on: ubuntu-latest
     permissions:
       contents: read
     strategy:
       matrix:
         node-version: [14.x, 16.x, 18.x]
+        os: [ubuntu-latest, windows-latest]
+        exclude:
+          - os: windows-latest
+            node-version: 14.x
```
Comment (simoneb) [inline .github/workflows/ci.yml:14]: exclude it altogether please, we don't need to support node 14 anymore

Comment (mcollina) [reply .github/workflows/ci.yml:14]: That'd be a semver-major bump. I don't mind doing so in this module, maybe we can drop v16 too.

**Comment by simoneb** [comment]
Looks like 16 is failing too now?

**Review by simoneb — APPROVED** [review]

**Comment by simoneb** [comment]
Thanks for all the work @madtisa , I'm going to merge this now

**Pull request merged**